src/layer/AbstractSentinelHubV1OrV2Layer.ts

```diff
@@ -64,12 +73,17 @@ export class AbstractSentinelHubV1OrV2Layer extends AbstractLayer {
   }
 
   protected getWmsGetMapUrlAdditionalParameters(): Record<string, any> {
+    let additionalParameters: Record<string, any> = {};
     if (this.mosaickingOrder) {
-      return {
-        priority: this.mosaickingOrder,
-      };
+      additionalParameters.priority = this.mosaickingOrder;
     }
-    return {};
+    if (this.upsampling) {
+      additionalParameters.upsampling = this.upsampling;
+    }
+    if (this.downsampling) {
+      additionalParameters.priority = this.downsampling;
```

Comment:
Typo: Should be `downsampling` instead of `priority `

Reply:
Was changed.

Reply:
Resolving.